### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by ddhodge**
Added examples to Explore>Security (#8663)

### CODE REVIEW

File: docs/content/latest/secure/authentication/ldap-authentication.md [hunk 1]
```diff
@@ -219,49 +207,38 @@ To use LDAP password authentication on a new YugabyteDB cluster, follow these st
...
-
-
-    ```
+    ```output
     # This is an autogenerated file, do not edit manually!
     host all yugabyte 127.0.0.1/0 trust
     host   all         all      0.0.0.0/0  ldap ldapserver=ldap.forumsys.com ldapprefix="uid=" ldapsuffix=", dc=example, dc=com" ldapport=389
     ```
 
+5. Configure database role(s) for the LDAP user(s).
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/secure/authentication/ldap-authentication.md:218]: fixed

File: docs/content/latest/secure/authentication/ldap-authentication.md [hunk 2]
```diff
@@ -219,49 +207,38 @@ To use LDAP password authentication on a new YugabyteDB cluster, follow these st
...
-
-
-```
-  We are creating a ROLE for username riemann supported by the test LDAP server. 
-
-  yugabyte=# CREATE ROLE riemann WITH LOGIN;
-  yugabyte=# GRANT ALL ON DATABASE yugabyte T0 riemann;
-
-```
+    We are creating a ROLE for username riemann supported by the test LDAP server.
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/secure/authentication/ldap-authentication.md:220]: fixed

**Review by lizayugabyte — COMMENTED** [review]
LGTM

**Comment by polarweasel** [comment]
In the markdown, it looks like you're trying to add content indented below a couple of these bullets. Make the indents four spaces to accomplish that; otherwise, it just renders as a single line.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /latest/explore/security/security/
Browser:   Safari 14.1 on Mac OS 10.15.7
Viewport:  1377 x 767 @2x
Language:  en-ca
Cookies:   Disabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-8694--infallible-bardeen-164bc9.netlify.app%2Flatest%2Fexplore%2Fsecurity%2Fsecurity%2F&os=OS+X&os_version=Catalina&browser=Safari&browser_version=14.0&resolution=1377x767&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-8694--infallible-bardeen-164bc9.netlify.app/latest/explore/security/security/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b4fe192c77a60008063cae&commentID=60b678ee0a4c2e00d0938bd2&resolution=resolved)

**Comment by polarweasel** [comment]
Hmm, that was supposed to be a cropped screenshot... oh well
<details><summary>Browser metadata</summary>





```
Path:      /latest/explore/security/security/
Browser:   Safari 14.1 on Mac OS 10.15.7
Viewport:  1377 x 767 @2x
Language:  en-ca
Cookies:   Disabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-8694--infallible-bardeen-164bc9.netlify.app%2Flatest%2Fexplore%2Fsecurity%2Fsecurity%2F&os=OS+X&os_version=Catalina&browser=Safari&browser_version=14.0&resolution=1377x767&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-8694--infallible-bardeen-164bc9.netlify.app/latest/explore/security/security/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b4fe192c77a60008063cae&commentID=60b67910587697009ab2543c&resolution=resolved)

**Comment by polarweasel** [comment]
There we go:

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /latest/explore/security/security/
Browser:   Safari 14.1 on Mac OS 10.15.7
Viewport:  1377 x 767 @2x
Language:  en-ca
Cookies:   Disabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-8694--infallible-bardeen-164bc9.netlify.app%2Flatest%2Fexplore%2Fsecurity%2Fsecurity%2F&os=OS+X&os_version=Catalina&browser=Safari&browser_version=14.0&resolution=1377x767&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-8694--infallible-bardeen-164bc9.netlify.app/latest/explore/security/security/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b4fe192c77a60008063cae&commentID=60b6793745165a00e4902431&resolution=resolved)

**Review by polarweasel — APPROVED** [review]
LGTM

**Pull request merged**